docs/GettingStartedDocs/VisualStudioWindows.md

```diff
@@ -24,7 +24,7 @@ You will need the following:
 - Intel's PSW2.6. This should be automatically installed on Windows 10 systems that support SGX.  If manual install is needed:
   - For systems with support for SGX1: [Intel's PSW 2.6,Intel Enclave Common API library](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md)
   - For systems with support for SGX1+FLC: [Intel's PSW2.6, Intel's Data Center Attestation Primitives and related dependencies](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md)
-- [Clang/LLVM for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe).  The installer will ask whether LLVM should be added to the PATH, and this must be done at least for the current user, so Visual Studio can find it.
+- [Clang/LLVM for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe).  The installer will ask whether LLVM should be added to the PATH, and this must be done at least for the current user, so Visual Studio can find it.
```

Comment:
@JeromeCustodio Thanks for this contribution. I would like to say that Ci/Cd tests clang-7. So ideally, we would like to tell developers to install clang-7. 
Could you please provide details on what issues you ran into with clang-7.

Reply:
hi @radhikaj, actually there's, no issue with clang-7.  it was working fine,  It's sociability with some single header library files I am using (nlohmann::json) which works better on clang-10.

Reply:
@JeromeCustodio In this case, is it necessary for this change? Given that our stabilized support is still based on clang-7, even the document change would create inconsistency to that.

Reply:
@mingweishih , I suppose the change is not required.  I haven't seen any issues using clang 10, will continue to use this for now. 
Thank you

Reply:
Thanks again for using the OE SDK and the contribution. We're still in the process of evaluating/updating our compiler support (meaning that all our CI/CD should pass with the newer version of the compiler), which would take a while. Before the process is done, we cannot officially claim that the new version of compiler is supported (even though it's actually working as in your case).